tutorials/best_practices/godot_notifications.rst

```diff
@@ -164,28 +164,36 @@ _init vs. initialization vs. export
 -----------------------------------
 
 If the script initializes its own node subtree, without a scene,
-that code should execute here. Other property or SceneTree-independent
-initializations should also run here. This triggers before ``_ready()`` or
-``_enter_tree()``, but after a script creates and initializes its properties.
+that code should execute in ``_init()``. Other property or SceneTree-independent
+initializations should also run here.
 
-Scripts have three types of property assignments that can occur during
-instantiation:
+``_init()`` triggers before ``_enter_tree()`` or ``_ready()``, but after a script
```

Comment:
The new text mentions `_init` so we may want to add a note for C# users. In C#, the `_init` method doesn't exist and instead users should use the regular class constructor.

The C# constructor will also execute before any of the engine callbacks (`_EnterTree`, `_Ready`, etc.) and after what's being referred to here as **Initial value assignment**.

Reply:
OK, adding a note that C#'s `_init()` equivalent is the constructor.